R/quantity.R

```diff
@@ -1,8 +1,11 @@
 #' @title Quantity
 #' @docType class
 #' @description  A quantity of the model (with unit, value) such as a Parameter or an Amount
-#'
-#' @field value The value of the quantity
+#' @field value The value of the quantity in unit
+#' @field unit The base unit in which the quantity is defined (Read-Only)
+#' @field dimension The dimension in which the quantity is defined  (Read-Only)
+#' @field persitable A boolean indicating whether a value will be defined in the simulated results for this quantity

```

Comment:
This is confusing for me.... Maybe

"A boolean indicating whether the ouput for this quantity will be available in simulated results"?

Reply:
sure. Updating